presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraIntergrationMixedCase.java

```diff
@@ -0,0 +1,167 @@
+        assertTrue(getQueryRunner().tableExists(session, "Test_CreateAs_Mixed_Join"));
+
+        assertUpdate(session, "DROP TABLE IF EXISTS TEST_CREATEAS");
+        assertFalse(getQueryRunner().tableExists(session, "TEST_CREATEAS"));
+
+        assertUpdate(session, "DROP TABLE IF EXISTS test_createas");
+        assertFalse(getQueryRunner().tableExists(session, "test_createas"));
+
+        assertUpdate(session, "DROP TABLE IF EXISTS Test_CreateAs_Mixed_Join");
+        assertFalse(getQueryRunner().tableExists(session, "Test_CreateAs_Mixed_Join"));
+    }
+
+    @Test
+    public void testDuplicatedColumNameCreateTable()
+    {
+        Session session = testSessionBuilder()
+                .setCatalog("cassandra")
+                .setSchema(KEYSPACE)
+                .build();
+
+        getQueryRunner().execute(session, "CREATE TABLE test (a integer, A integer)");
+        assertTrue(getQueryRunner().tableExists(session, "test"));
+
+        getQueryRunner().execute(session, "CREATE TABLE TEST (a integer, A integer)");
+        assertTrue(getQueryRunner().tableExists(session, "TEST"));
+
+        assertQueryFails("CREATE TABLE Test (a integer, a integer)", "line 1:31: Column name 'a' specified more than once");
+        assertFalse(getQueryRunner().tableExists(session, "Test"));
+
+        assertUpdate(session, "DROP TABLE IF EXISTS test");
+        assertFalse(getQueryRunner().tableExists(session, "test"));
+
+        assertUpdate(session, "DROP TABLE IF EXISTS TEST");
+        assertFalse(getQueryRunner().tableExists(session, "TEST"));
+    }
+
+    @Test
+    public void testSelect()
+    {
+        Session session = testSessionBuilder()
+                .setCatalog("cassandra")
+                .setSchema(KEYSPACE)
+                .build();
+
+        getQueryRunner().execute(session, "CREATE TABLE Test_Select  AS SELECT * FROM tpch.region where regionkey=3");
+        assertTrue(getQueryRunner().tableExists(session, "Test_Select"));
+        assertQuery("SELECT * from test_connetor.Test_Select", "VALUES (3, 'EUROPE', 'ly final courts cajole furiously final excuse')");
+
+        getQueryRunner().execute(session, "CREATE TABLE test_select  AS SELECT * FROM tpch.region LIMIT 2");
+        assertQuery("SELECT COUNT(*) FROM test_connetor.test_select", "VALUES 2");
+
+        getQueryRunner().execute(session, "DROP TABLE IF EXISTS TEST_SELECT");
+        getQueryRunner().execute(session, "DROP TABLE IF EXISTS Test_Select");
+    }
+
+    @Test
+    public void testShowSchemas()
+    {
+        session.execute("CREATE KEYSPACE test_keyspace WITH REPLICATION = {'class':'SimpleStrategy', 'replication_factor': 1}");
+        assertContainsEventually(() -> getQueryRunner().execute("SHOW SCHEMAS FROM cassandra"), resultBuilder(getSession(), createUnboundedVarcharType())
+                .row("test_keyspace")
+                .build(), new Duration(1, MINUTES));
```

Comment:
Lets add schema creation for same KEYSPACE in upper case as well and then check for lowercase and upper cae for in testShowSchemas